clients/go/test/grpc-app/main.go

```diff
@@ -65,22 +62,25 @@ func realMain() (outErr error) {
 	}
 	defer shutdown()
 
+	// set Justification config via environment variables.
+	// can't set it via yaml file because we don't know pubKeyEndpoint when we write yaml file.
+	if err := os.Setenv("AUDIT_CLIENT_JUSTIFICATION_PUBLIC_KEYS_ENDPOINT", pubKeyEndpoint); err != nil {
+		return err
+	}
+	if err := os.Setenv("AUDIT_CLIENT_JUSTIFICATION_ENABLED", "true"); err != nil {
+		return err
+	}
```

Comment:
Nit: but we can set this value in config file, right?

Reply:
`enabled` can be set in the config file, but do we want to separate the justification setting of `endpoint` and `enabled`? I feel it is a little bit confusing.

Reply:
I think we can set a fake endpoint in the config file and still override it here.

Reply:
done